app/jobs/add_registration_job.rb

```diff
@@ -2,7 +2,9 @@
 
 class AddRegistrationJob < ApplicationJob
   before_enqueue do |job|
-    Rails.cache.write("#{job.arguments[1]}-#{job.arguments[2]}-processing", true)
+    competition_id = job.arguments[1]
+    user_id = job.arguments[2]
```

Comment:
Ruby 101: `competition_id, user_id = job.arguments`

Reply:
Ruby 102, arrays begin at 0 :P and I don't think `_,competition_id, user_id = job.arguments` looks that great?

Reply:
Alternatively I could change the order of the job params

Reply:
Ruby 103: `competition_id, user_id = job.arguments.drop(1)`

Reply:
And to be honest your underscore approach isn't too bad either.

Reply:
ok, if you fine with that, i changed it to that